.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-  - "8"
-  - "10"
+  - "12"
```

Comment:
This will require a major version bump.

Reply:
Yes as I did this in the package.json.

Reply:
I'm not sure what you mean. The version in package.json is still `>= 8`. It's likely we should update that to `>= 12`.